Automattic/wp-calypso
# A/B test: make coupon field more visible vs keep is as now

**Test plan:**

* Add yourself to `control` group of `couponCodeMoreProminent` test
* Confirm that order summary looks and works as in production, with coupon field being available after clicking on a link:

<img width="273" alt="zrzut ekranu 2018-06-08 o 11 57 31" src="https://user-images.githubusercontent.com/205419/41152184-340dbd02-6b13-11e8-8a27-925936fee901.png">
<img width="274" alt="zrzut ekranu 2018-06-08 o 11 57 35" src="https://user-images.githubusercontent.com/205419/41152185-342ccc42-6b13-11e8-83f8-e3fa4fbae7cc.png">

* Check in responsive mode as well:

<img width="473" alt="zrzut ekranu 2018-06-08 o 11 58 32" src="https://user-images.githubusercontent.com/205419/41152215-49eaf36a-6b13-11e8-874c-6ad61afc4d15.png">
<img width="470" alt="zrzut ekranu 2018-06-08 o 11 58 44" src="https://user-images.githubusercontent.com/205419/41152225-4e68bf4e-6b13-11e8-9636-baf40e6e6379.png">


* Add yourself to `prominent` group of `couponCodeMoreProminent` test
* Confirm that order summary looks like this:

<img width="296" alt="zrzut ekranu 2018-06-08 o 12 04 02" src="https://user-images.githubusercontent.com/205419/41152515-19c3704e-6b14-11e8-8b39-2593696416ff.png">

* Enter a coupon, confirm it now looks like this:

<img width="277" alt="zrzut ekranu 2018-06-05 o 11 38 09" src="https://user-images.githubusercontent.com/205419/40968073-f6fd5798-68b4-11e8-8669-29f4f2ae4d53.png">

* Check in responsive mode too:

<img width="486" alt="zrzut ekranu 2018-06-08 o 12 03 24" src="https://user-images.githubusercontent.com/205419/41152471-f5c28964-6b13-11e8-8840-298b063ab469.png">
